lib/bsdlib/nrf91_sockets.c

```diff
@@ -1074,6 +1078,8 @@ static int nrf91_socket_offload_getaddrinfo(const char *node,
 	}
 	nrf_freeaddrinfo(nrf_res);
```

Comment:
Should the happen inside the "error:" label? It is not in the original file either, but are we guaranteed that the memory is freed automatically in bsdlib on all errors? @evenl ?

Reply:
@evenl Have you had the time to look at this?

Reply:
No memory has been allocated if the getaddrinfo call returns an error, so I guess not.